### ISSUE TITLE
Windows become black on 2nd monitor and cursor glitch

### ISSUE THREAD

**Issue by Iss-in** [WillPower3309/swayfx#105]
- **Swayfx Version:**
 sway version 0.1.1-9e690126 (Jan 22 2023, branch 'master')

- **Debug Log:**
 https://gist.github.com/Iss-in/45b300c3ec4a388a8461a9844e436ae4

- **Configuration File:**
attached in gist

- **Description:**
  - The steps you took in plain English to reproduce the problem.

- open a new window on current monitor 
- shift it to 2nd monitor 
- window becomes all black
- bring it back to current monitor, its fine again.

sometimes i also got this cursor glitch

![2023-01-22_21-41-50](https://user-images.githubusercontent.com/36916613/213926880-7d3da14d-33c3-4366-998e-81a86d3c8411.png)
![2023-01-22_21-54-10](https://user-images.githubusercontent.com/36916613/213926938-e040aebe-1506-4a81-aa87-fc022225792e.png)
2nd screenshot has firefox in ist monitor, but rendered all black

**Comment by WillPower3309** [comment]
@ErikReider potential dim issue?

**Comment by Iss-in** [comment]
so ist screen is my laptop, while 2nd is external monitor. after i detached my ext screen , isse persist on laptop even after rebooting. 

restarting/exiting/rebooting sway did nothing. only way it was solved was running sway on a different tty

**Comment by ErikReider** [comment]
> @ErikReider potential dim issue?

Shouldn't be. Very strange... Does sway 1.8 work as expected?

**Comment by Iss-in** [comment]
yes, both 1.8 and release version work normally

**Comment by WillPower3309** [comment]
Would you be able to try the `-git` package?

**Comment by Iss-in** [comment]
oh sorry i didnt list it , im using the swayfx-git package only, for shadows

**Comment by WillPower3309** [comment]
Hey @Iss-in , are you still able to reproduce this on the latest swayfx-git?

**Comment by Iss-in** [comment]
hey, sorry this issue was happening with my laptop and external monitor, sadly i no longer have that laptop. now its just monitors and rig, and havent seen it happening since then

**Comment by WillPower3309** [comment]
Weird! I haven't been able to reproduce so I think I'll close for now :S
